server/src/main/scala/Routes.scala

```diff
@@ -0,0 +1,18 @@
+package server
+
+import akka.http.scaladsl.server.Directives
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/olivier-blanvillain/19x19-net/file/2594775828/issues/source?bid=3301982&fileBranchId=3306761#l3)